cargo-shuttle/src/lib.rs

```diff
@@ -210,6 +213,53 @@ impl Shuttle {
         .context("failed to deploy cargo project")
     }
 
+    async fn check_lib_version(&self) -> Result<()> {
+        let cargo_tree = PCmd::new("cargo")
```

Comment:
PR #192 uses `toml-edit` to [set the shuttle service version](https://github.com/shuttle-hq/shuttle/pull/192/files#diff-7fea87037f7c8fa590c68cd9e295ce99aa94dca3eab2be9a935dfda0a90cd5d0R111). I wonder if we can use the same library to get the version and make this easier?